src/Modal/AlertModal.test.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import AlertModal from './AlertModal';
+
+/* eslint-disable react/prop-types */
+jest.mock('./Portal', () => function PortalMock(props) {
+  const { children, ...otherProps } = props;
+  return (
+    <paragon-portal {...otherProps}>
+      {children}
+    </paragon-portal>
+  );
+});
+
+jest.mock('react-focus-on', () => ({
+  FocusOn: (props) => {
+    const { children, ...otherProps } = props;
+    return (
+      <focus-on {...otherProps}>{children}</focus-on>
+    );
+  },
+}));
+
+function Body() {
+  return (
+    <div>
+      The body of alert.
+    </div>
+  );
+}
+
+describe('<AlertModal />', () => {
+  const isOpen = true;
+  const closeFn = jest.fn();
+  describe('when isOpen', () => {
+    const wrapper = shallow((
+      <AlertModal
+        title='some title'
```

Comment:
Check the CI on this PR; its failing due to some linting issues, one of which is the use of single quotes here.